ui/js/controllers/perf/graphs.js

```diff
@@ -486,7 +486,17 @@ perf.controller('GraphsCtrl', [
                 });
 
                 //Closes popup on-click of "X"
-                $('#close-popup').bind("click", plotUnselected);
+                $('#close-popup').bind("click", hideTooltip);
+
+                //Checks for clicks outside element data-display and closes popup
```

Comment:
That is applicable for the latter part, the binding for #close-popup is for the corner "x" button